xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1398,7 +1398,14 @@ bool CRenderManager::GetStats(int &lateframes, double &pts, int &queued, int &di
 
 void CRenderManager::CheckEnableClockSync()
 {
-  if (fabs(m_fps - g_graphicsContext.GetFPS()) < 0.01)
+  // refresh rate can be a multiple of video fps
+  double diff;
+  if (g_graphicsContext.GetFPS() >= m_fps)
+    diff = fmod(g_graphicsContext.GetFPS(), m_fps);
```

Comment:
this is fine adjustment for a/v sync
you can't calculate an offset for pts/vblank, if fps does not exactly match refresh rate.
video pts relates to the time a frame is displayed (vblank) and ideally audio pts is synced exactly to this time.